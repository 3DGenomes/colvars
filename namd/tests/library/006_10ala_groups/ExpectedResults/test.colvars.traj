# step         rmsdA                 fs_rmsdA               rmsdB                 fs_rmsdB               
           0    4.84870372257743e-01  0.00000000000000e+00   4.84870372257743e-01  0.00000000000000e+00  
           1    4.84397558390947e-01  2.83367585575090e+01   4.84397558390947e-01  2.83367585575090e+01  
           2    4.83907428259583e-01  2.23323979521607e+01   4.83907428259583e-01  2.23323979521607e+01  
           3    4.83522392476985e-01  1.53955610554231e+01   4.83522392476985e-01  1.53955610554231e+01  
           4    4.83083112856857e-01  9.99342571740182e+00   4.83083112856857e-01  9.99342571740182e+00  
           5    4.82621016610959e-01  7.09302308657841e+00   4.82621016610959e-01  7.09302308657841e+00  
           6    4.82181848616375e-01  3.91166594608564e+00   4.82181848616375e-01  3.91166594608564e+00  
           7    4.81645520125956e-01  4.24164227099853e+00   4.81645520125956e-01  4.24164227099853e+00  
           8    4.81136761645330e-01  7.89108784546697e+00   4.81136761645330e-01  7.89108784546697e+00  
           9    4.80598465150394e-01  1.37277483116144e+01   4.80598465150394e-01  1.37277483116144e+01  
          10    4.80094226695462e-01  2.35144233534901e+01   4.80094226695462e-01  2.35144233534901e+01  
          11    4.79704703208600e-01  3.41354943015590e+01   4.79704703208600e-01  3.41354943015590e+01  
          12    4.79506792183814e-01  4.13351392590237e+01   4.79506792183814e-01  4.13351392590237e+01  
          13    4.79327275850326e-01  4.60694618560190e+01   4.79327275850326e-01  4.60694618560190e+01  
          14    4.79314968874545e-01  5.09721600038501e+01   4.79314968874545e-01  5.09721600038501e+01  
          15    4.79397379647665e-01  4.98906215300851e+01   4.79397379647665e-01  4.98906215300851e+01  
          16    4.79394129016958e-01  4.77981934055389e+01   4.79394129016958e-01  4.77981934055389e+01  
          17    4.79404085661515e-01  4.38493340530944e+01   4.79404085661515e-01  4.38493340530944e+01  
          18    4.79306164127760e-01  3.93335566909782e+01   4.79306164127760e-01  3.93335566909782e+01  
          19    4.79258291379496e-01  3.54949935873750e+01   4.79258291379496e-01  3.54949935873750e+01  
          20    4.79299937828241e-01  2.96144905452134e+01   4.79299937828241e-01  2.96144905452134e+01  
