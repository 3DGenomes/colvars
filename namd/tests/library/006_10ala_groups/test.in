colvarsTrajFrequency    1

indexFile ../Common/da.ndx


colvar {
    name rmsdA

    outputSystemForce on
    #  outputAppliedForce on
    width 1.0 

    lowerboundary 0.0
    upperboundary 5.0

    lowerwallconstant 100.0
    upperwallconstant 100.0

    rmsd {
        atoms {
            indexGroup Protein_C-alpha
        }
        refPositionsFile ../Common/da.xyz
    }
} 


colvar {
    name rmsdB

    outputSystemForce on
    #  outputAppliedForce on
    width 1.0 

    lowerboundary 0.0
    upperboundary 5.0

    lowerwallconstant 100.0
    upperwallconstant 100.0

    rmsd {
        atoms {
            indexGroup Protein_C-alpha
        }
        refpositionsfile ../Common/da.xyz
    }
} 


harmonic {
    colvars        rmsdA rmsdB
    forceConstant  1.0
    centers        0.0   0.0
}
