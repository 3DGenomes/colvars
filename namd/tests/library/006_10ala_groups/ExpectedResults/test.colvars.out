colvars: ----------------------------------------------------------------------
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # indexFile = index.ndx
colvars: The following index groups were read from the index file "index.ndx":
colvars:   System (104 atoms).
colvars:   Protein (104 atoms).
colvars:   Protein-H (51 atoms).
colvars:   C-alpha (10 atoms).
colvars: # analysis = off [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 2000
colvars: # colvarsTrajAppend = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = rmsdA
colvars:   Initializing a new "rmsd" component.
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:       Initializing atom group "atoms".
colvars:       # indexGroup = C-alpha
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on [default]
colvars:       Atom group "atoms" defined, 10 atoms initialized: total mass = 120.11, total charge = 0.53.
colvars:     # refPositions =  [default]
colvars:     # refPositionsFile = ../Common/da.pdb
colvars:     # refPositionsCol = "" [default]
colvars:     Enabling "centerReference" and "rotateReference", to minimize RMSD before calculating it as a variable: if this is not the desired behavior, disable them explicitly within the "atoms" block.
colvars:     This is a standard minimum RMSD, derivatives of the optimal rotation will not be computed as they cancel out in the gradients.
colvars:   All components initialized.
colvars:   # width = 1
colvars:   # lowerBoundary = 0
colvars:   # lowerWallConstant = 100
colvars:   # lowerWall = 0 [default]
colvars:   # upperBoundary = 5
colvars:   # upperWallConstant = 100
colvars:   # upperWall = 5 [default]
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputSystemForce = on
colvars:   # outputAppliedForce = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = rmsdB
colvars:   Initializing a new "rmsd" component.
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:       Initializing atom group "atoms".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID = { BH }
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on [default]
colvars:       Atom group "atoms" defined, 10 atoms initialized: total mass = 120.11, total charge = 0.53.
colvars:     # refPositions =  [default]
colvars:     # refPositionsFile = ../Common/da.pdb
colvars:     # refPositionsCol = "" [default]
colvars:     Enabling "centerReference" and "rotateReference", to minimize RMSD before calculating it as a variable: if this is not the desired behavior, disable them explicitly within the "atoms" block.
colvars:     This is a standard minimum RMSD, derivatives of the optimal rotation will not be computed as they cancel out in the gradients.
colvars:   All components initialized.
colvars:   # width = 1
colvars:   # lowerBoundary = 0
colvars:   # lowerWallConstant = 100
colvars:   # lowerWall = 0 [default]
colvars:   # upperBoundary = 5
colvars:   # upperWallConstant = 100
colvars:   # upperWall = 5 [default]
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputSystemForce = on
colvars:   # outputAppliedForce = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 2 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonic" instance.
colvars:   # name = "harmonic1" [default]
colvars:   # colvars = { rmsdA, rmsdB }
colvars:   # outputEnergy = off [default]
colvars:   # forceConstant = 1
colvars:   # centers = { 0, 0 }
colvars:   # targetCenters = { 0, 0 } [default]
colvars:   # targetForceConstant = 0 [default]
colvars:   # outputCenters = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: The final output state file will be "test.colvars.state".
colvars: Writing to colvar trajectory file "test.colvars.traj".
colvars: Saving collective variables state to "test.colvars.state".
